test/libsolidity/syntaxTests/abiEncoder/abi_encode_call_tuple_from_function.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+    function g0() internal {}
+    function g1() internal returns (uint) {}
+    function g2() internal returns (uint, uint) {}
+
+    function f() public pure {
+        abi.encodeCall(this.f, g0());
+        abi.encodeCall(this.f, g1());
+        abi.encodeCall(this.f, g2());
```

Comment:
I used `this.f` in my original example for brevity but in the test let's use proper functions that actually have the number or args matching number of returns in `g0()`, `g1()` and `g2()`. I.e. let's make it so that the error we're interested in is the only error in the snippet and it would compile otherwise.